sermon-mount.txt

```diff
@@ -1,215 +1,215 @@
 
-Ask, and it shall be given you; seek, and ye shall find; knock, and it shall be opened unto you:
+Ask, and it shall be given unto you; seek, and ye shall find; knock, and it shall be opened unto you.
 
-For every one that asketh receiveth; and he that seeketh findeth; and to him that knocketh it shall be opened.
+For every one that asketh, receiveth; and he that seeketh, findeth; and to him that knocketh, it shall be opened.
 
-Or what man is there of you, whom if his son ask bread, will he give him a stone?
+Or what man is there of you, who, if his son ask bread, will give him a stone?
 
 Or if he ask a fish, will he give him a serpent?
 
-If ye then, being evil, know how to give good gifts unto your children, how much more shall your Father which is in heaven give good things to them that ask him?
+If ye then, being evil, know how to give good gifts unto your children, how much more shall your Father who is in heaven give good things to them that ask him?
 
-Therefore all things whatsoever ye would that men should do to you, do ye even so to them: for this is the law and the prophets.
+Therefore, all things whatsoever ye would that men should do to you, do ye even so to them, for this is the law and the prophets.
 
-Enter ye in at the strait gate: for wide is the gate, and broad is the way, that leadeth to destruction, and many there be which go in thereat:
+Enter ye in at the strait gate; for wide is the gate, and broad is the way, which leadeth to destruction, and many there be who go in thereat;
 
 Because strait is the gate, and narrow is the way, which leadeth unto life, and few there be that find it.
 
-Beware of false prophets, which come to you in sheep’s clothing, but inwardly they are ravening wolves.
+Beware of false prophets, who come to you in sheep’s clothing, but inwardly they are ravening wolves.
 
 Ye shall know them by their fruits. Do men gather grapes of thorns, or figs of thistles?
 
 Even so every good tree bringeth forth good fruit; but a corrupt tree bringeth forth evil fruit.
 
-A good tree cannot bring forth evil fruit, neither can a corrupt tree bring forth good fruit.
+A good tree cannot bring forth evil fruit, neither a corrupt tree bring forth good fruit.
 
 Every tree that bringeth not forth good fruit is hewn down, and cast into the fire.
 
-Wherefore by their fruits ye shall know them.
-
-Not every one that saith unto me, Lord, Lord, shall enter into the kingdom of heaven; but he that doeth the will of my Father which is in heaven.
-
-Many will say to me in that day, Lord, Lord, have we not prophesied in thy name? and in thy name have cast out devils? and in thy name done many wonderful works?
+Wherefore, by their fruits ye shall know them.
 
-And then will I profess unto them, I never knew you: depart from me, ye that work iniquity.
+Not every one that saith unto me, Lord, Lord, shall enter into the kingdom of heaven; but he that doeth the will of my Father who is in heaven.
```

Comment:
this is not actually a difference... the comparison tool isn't working well on this